corehq/apps/dump_reload/couch/dump.py

```diff
@@ -88,8 +89,8 @@ class ToggleDumper(DataDumper):
 
     def dump(self, output_stream):
         count = 0
-        user_ids = _user_ids_in_domain(self.domain)
-        for toggle in _get_toggles_to_migrate(self.domain, user_ids):
+        usernames = get_all_usernames_by_domain(self.domain)
```

Comment:
as far as can tell this has always been broken. The unit test works because it's using mocking.